### ISSUE TITLE
Change Background Color For Better Readabilty

### ISSUE THREAD

**Issue by DivyanshuSonwal** [keptn/keptn.github.io#1611]

**Comment by thisthat** [comment]
Hey @DivyanshuSonwal, I moved the issue to the right repository! 
Sure think, I am assigning it to you! Thanks a lot 🙌

**Comment by DivyanshuSonwal** [comment]
Thank you😄.

**Comment by mowies** [comment]
@DivyanshuSonwal can you post a link to where you found the issue please?

**Comment by DivyanshuSonwal** [comment]
> @DivyanshuSonwal can you post a link to where you found the issue please?
Yes, sure. Here are two examples, and there are many more. I am working on it, and as soon I update all the pages, I will create a PR for that.
https://keptn.sh/docs/concepts/automated_operations/
https://keptn.sh/docs/concepts/automated_operations/
